source/dub/dependency.d

```diff
@@ -34,6 +34,62 @@ struct PackageDependency {
 	Dependency spec;
 }
 
+struct GitDep{
```

Comment:
Style: Should be fully spelled out (`GitDependency`) and before `{` there should always be a single space (or a line break in case of functions).